<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #cccccc;
border-left: 4px solid #0A1172;
border-right: 4px solid #0A1172;
border-bottom: 4px solid #0A1172;</string>
  </property>
  <widget class="QLineEdit" name="passwordText">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid #0A1172;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid #0A1172;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailText">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid #0A1172;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid #0A1172;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameText">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid #0A1172;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="email">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid #0A1172</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
color: #0a1172;
</string>
   </property>
   <property name="text">
    <string>Add User</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid #0A1172;</string>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUser">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 5px;
background-color: #0A1172;
color: white;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="role">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid #0A1172;
</string>
   </property>
   <item>
    <property name="text">
     <string>-- Select Role --</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
